clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -416,6 +438,14 @@ public void send(String method, Object... args) {
         sendHubMessage(invocationMessage);
     }
 
+    /**
+     * Invokes a hub method on the server using the specified name and arguments.
+     * @param returnType The expected return type.
```

Comment:
Suggest: "The return type of the server method."

(Same as above, align with .NET docs)